examples/pytorch_lightning_simple.py

```diff
@@ -0,0 +1,190 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using PyTorch Lightning.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+PyTorch Lightning, and MNIST. We optimize the neural network architecture as well as the optimizer
+configuration. As it is too time consuming to use the whole MNIST dataset, we here use a small
+subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python pytorch_lightning_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize pytorch_lightning_simple.py objective --n-trials=100 --study \
+      $STUDY_NAME --storage sqlite:///example.db
+
+"""
+
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import pytorch_lightning as pl
```

Comment:
I think `pytorch_lightning` is a new requirement for example jobs of CI.
Please add `pytorch-lightning` to [`setup.py`](https://github.com/pfnet/optuna/blob/master/setup.py#L61-L68).

Reply:
Thanks, it was also pointed out by @crcrpar. It should be fixed now.